checkov/common/graph/graph_builder/graph_components/blocks.py

```diff
@@ -44,7 +44,10 @@ def _extract_inner_attributes(self) -> Dict[str, Any]:
             if isinstance(attribute_value, dict) or (
                     isinstance(attribute_value, list) and len(attribute_value) > 0 and isinstance(attribute_value[0],
                                                                                                   dict)):
-                inner_attributes = get_inner_attributes(attribute_key, attribute_value)
+                inner_attributes = self.get_inner_attributes(
+                    attribute_key=attribute_key,
+                    attribute_value=attribute_value,
+                )
```

Comment:
@gruebel Unnecessary change?

Reply:
before `get_inner_attributes` was a function inside the file, now it is a `@classmethod` that's why there is now a `self.` at the beginning and it is reformated, because I implemented it differently before and them came up with a better approach 😄